From e924f6d8a1d2245c4fa55c0f5af0fbb5e29503f2 Mon Sep 17 00:00:00 2001
From: Ming Wang <wangming01@loongson.cn>
Date: Thu, 28 Dec 2023 19:46:34 +0800
Subject: [PATCH 10/12] LoongArch64: Add LoongArch64 architecture support
 information

Add LoongArch64 architecture support information to the README and
help.c files.

Co-developed-by: Youling Tang <tangyouling@loongson.cn>
Signed-off-by: Youling Tang <tangyouling@loongson.cn>
Signed-off-by: Ming Wang <wangming01@loongson.cn>
Signed-off-by: Lianbo Jiang <lijiang@redhat.com>
---
 README | 6 ++++--
 help.c | 6 ++++--
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/README b/README
index 6ecdd10aa126..80cc7b166703 100644
--- a/README
+++ b/README
@@ -37,8 +37,8 @@
   These are the current prerequisites: 
 
   o  At this point, x86, ia64, x86_64, ppc64, ppc, arm, arm64, alpha, mips,
-     mips64, riscv64, s390 and s390x-based kernels are supported.  Other
-     architectures may be addressed in the future.
+     mips64, loongarch64, riscv64, s390 and s390x-based kernels are supported.
+     Other architectures may be addressed in the future.
 
   o  One size fits all -- the utility can be run on any Linux kernel version
      version dating back to 2.2.5-15.  A primary design goal is to always
@@ -100,6 +100,8 @@
      ppc64le dumpfiles may be built by typing "make target=PPC64".
   o  On an x86_64 host, an x86_64 binary that can be used to analyze
      riscv64 dumpfiles may be built by typing "make target=RISCV64".
+  o  On an x86_64 host, an x86_64 binary that can be used to analyze
+     loongarch64 dumpfiles may be built by typing "make target=LOONGARCH64".
 
   Traditionally when vmcores are compressed via the makedumpfile(8) facility
   the libz compression library is used, and by default the crash utility
diff --git a/help.c b/help.c
index 5feb43d1d727..623b09cc3517 100644
--- a/help.c
+++ b/help.c
@@ -9559,8 +9559,8 @@ char *README[] = {
 "  These are the current prerequisites: ",
 "",
 "  o  At this point, x86, ia64, x86_64, ppc64, ppc, arm, arm64, alpha, mips,",
-"     mips64, riscv64, s390 and s390x-based kernels are supported.  Other",
-"     architectures may be addressed in the future.",
+"     mips64, loongarch64, riscv64, s390 and s390x-based kernels are supported.",
+"     Other architectures may be addressed in the future.",
 "",
 "  o  One size fits all -- the utility can be run on any Linux kernel version",
 "     version dating back to 2.2.5-15.  A primary design goal is to always",
@@ -9621,6 +9621,8 @@ README_ENTER_DIRECTORY,
 "     ppc64le dumpfiles may be built by typing \"make target=PPC64\".",
 "  o  On an x86_64 host, an x86_64 binary that can be used to analyze",
 "     riscv64 dumpfiles may be built by typing \"make target=RISCV64\".",
+"  o  On an x86_64 host, an x86_64 binary that can be used to analyze",
+"     loongarch64 dumpfiles may be built by typing \"make target=LOONGARCH64\".",
 "",
 "  Traditionally when vmcores are compressed via the makedumpfile(8) facility",
 "  the libz compression library is used, and by default the crash utility",
-- 
2.41.0

